src/Sulu/Component/Rest/ListBuilder/ListRepresentation.php

```diff
@@ -67,13 +67,23 @@ class ListRepresentation extends PaginatedRepresentation
     protected $total;
 
     /**
-     * @param mixed  $data       The data which will be presented
-     * @param string $rel        The name of the relation inside of the _embedded field
-     * @param string $route      The name of the route, for generating the links
-     * @param array  $parameters The parameters to append to the route
-     * @param int    $page       The number of the current page
-     * @param int    $limit      The size of one page
-     * @param int    $total      The total number of elements
+     * @var mixed[]
+     */
+    protected $data;
+
+    /**
+     * @var string
+     */
+    protected $rel;
```

Comment:
i use this now for the name of the file before it was only used by the wrapped object (see constructor)